ProjectCoimbra.UWP/Project.Coimbra/Helpers/SongPagesHelper.cs

```diff
@@ -185,5 +184,31 @@ public static async Task FillListBoxAsync(ListBox listBox, string selectedValue,
                 nextButton.IsEnabled = true;
             }
         }
+
+        /// <summary>
+        /// Removes Item from list
+        /// </summary>
+        /// <param name="key">item key</param>
+        public static async Task RemoveFile(string key)
+        {
+            if (string.IsNullOrWhiteSpace(key))
+            {
+                throw new ArgumentNullException(nameof(key));
+            }
+
+            var midiFolder = await GetMidiFolderAsync(false).ConfigureAwait(true);
+            if (midiFolder == null)
+            {
+                throw new ArgumentNullException(nameof(midiFolder));
+            }
+
+            var file = await midiFolder.GetFileAsync(key.Substring(key.LastIndexOf('\\') + 1));
```

Comment:
Can you check if these async methods work with `.ConfigureAwait(false)` added? If so, that will make the code more efficient. If not, could you add `.ConfigureAwait(true)` so that we know this can't be used here.